pkg/filetype/file_type.go

```diff
@@ -25,7 +25,7 @@ var JsonFileType = FileType{
 // Instance of the FileType object to
 // represent a YAML file
 var YamlFileType = FileType{
-	"json",
```

Comment:
Interesting. I would assumed the tests would have caught this bug.

Reply:
We don't use the Name attribute in the FileType struct anywhere for search or validate logic. The purprose was for display/metadata which is why this wasn't caught.